### PR TITLE
New Pull Request for EVA signature

### PR DESCRIPTION
**Pull request by DrySlime**
I now have checked the email, unset and set it again and i hope everything is set up properly now.

### CODE REVIEW

**Comment by DrySlime** [comment]
ECA works, YAAAY

File: README.md [hunk 1]
```diff
@@ -17,7 +17,6 @@ Name | Description
 * [SOME/IP Provider](https://github.com/eclipse-kuksa/kuksa-someip-provider)
 * [DDS Provider](https://github.com/eclipse-kuksa/kuksa-dds-provider)
 * [CSV Provider](https://github.com/eclipse-kuksa/kuksa-csv-provider)
-* [eCAL Provider](https://github.com/eclipse-kuksa/kuksa-incubation/tree/main/ecal2val)
```
Comment (erikbosch) [inline README.md:20]: I assume this is a merge conflict error, i.e. that it is not intentional to remove this line?

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 Name | Description
 ---- | -----------
 [Replay](./replay)             |[KUKSA Server](https://github.com/eclipse/kuksa.val/tree/master/kuksa-val-server) replay script for previously recorded files, created by running KUKSA Server with `--record` argument **Note: Replay is deprecated and will reach End-of-Life 2024-12-31!**
-[ecal2val](./ecal2val)             |Provider for eCAL data to [KUKSA Databroker](https://github.com/eclipse/kuksa.val/tree/master/kuksa_databroker)
+[fone2val](./fone2val)        | F1 Telemetrydata source for [KUKSA Databroker](https://github.com/eclipse/kuksa.val/tree/master/kuksa_databroker)
```
Comment (erikbosch) [inline README.md:10]: Here removing the ecal2val line is fine as I better should have removed it in last commit

**Review by erikbosch — APPROVED** [review]
Looks good to me.

There is some minor things which I believe is merge conflict regressions in main README but I do not see that as necessary as fix before merge as I anyway will need to update the README in the planned move of the content of this PR after merge to https://github.com/eclipse-kuksa/kuksa-incubation (as we are about to archive this repo)

**Review by SebastianSchildt — APPROVED** [review]
Then let's merge here. And "prepare for move" :)

Now I really need a PS...

**Pull request merged**